QualifiedTeachersApi/src/QualifiedTeachersApi/Jobs/Scheduling/RecurringJobsOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace QualifiedTeachersApi.Jobs.Scheduling;
+
+public class RecurringJobsOptions
+{ 
+    public required BatchSendQtsAwardedEmailsJobOptions BatchSendQtsAwardedEmails { get; init; }
+}
+
+public class BatchSendQtsAwardedEmailsJobOptions
+{
+    public required DateTime InitialLastAwardedToUtc { get; init; }
+    public required int EmailDelayDays { get; init; } = 3;
```

Comment:
I think it would be better to have these defaults in config to keep everything in one place.
```suggestion
    public required int EmailDelayDays { get; init; };
```

Reply:
Fixed